### PR TITLE
Automation Graph rules 

### PR DESCRIPTION
**Pull request by bmfmancini**
Updated Graph rules doc

### CODE REVIEW

**Comment by TheWitness** [comment]
Travis will be made at you, but I'll merge anyway.  Do you get Travis messages?

**Comment by bmfmancini** [comment]
Its been spinning for a while waiting for the ci build maybe they are down ?

**Comment by TheWitness** [comment]
No, they are dumping 'free' CI.  So, we've been making plans.  I'll merge, but it'll break.  Line lengths.

File: Device-Rules.md [hunk 1]
```diff
@@ -2,9 +2,24 @@
...
+Based on the device rule REGEX the automation engine can apply the appropriate template to the device i.e if there is a REGEX for IOS.
+The automation engine may pick the Cisco router template for the device
+
 ![Automation Device Rules](images/automation-device-templates.png)
+Below you will be able to choose a REGEX that will help identify the device the REGEX can be in any Perl supported format.
+You can select either to match by system description i.e the device hostname or the system name i.e Cisco
+
+An example regex for Cisco would be
+(Cisco Internetwork Operating System Software|IOS)
+This matches based on the system name of the Cisco IOS software
```
Comment (netniV) [inline Device-Rules.md:15]: Should this be "for any Cisco device or specifically a device with IOS in its name/description".

**Pull request merged**